com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1425,6 +1432,89 @@ private void configurePreBuildProjectOptions() throws IOException, CompileExcept
         }
     }
 
+    private void transpileLua(IProgress monitor) throws CompileExceptionError, IOException {
+        List<ILuaTranspiler> transpilers = PluginScanner.getOrCreatePlugins("com.defold.extension.pipeline", ILuaTranspiler.class);
```

Comment:
What if two different transpilers express that they'd like to transpile the same file extension? I think we need to emit an error or something. Both in Bob and in the editor.

Reply:
This looks fine to me, why shouldn't it work? I can imagine a custom transpilers for e.g. `.json` files that work in parallel and produce different lua configurations from different json files.

Reply:
Hmm, yes. That is a very good point.

I guess the issue is really if two transpilers emit two different versions of the output for the same output path. But that issue might not even be limited to transpilers. 🤔